packages/devtools/plugin/src/cli.ts

```diff
@@ -10,16 +10,25 @@ import {
   ROUTE_BASENAME,
 } from '@modern-js/devtools-kit';
 import { withQuery } from 'ufo';
-import { Options, resolveOptions } from './config';
+import { Options, InlineOptions, resolveOptions } from './config';
 import { setupClientConnection } from './rpc';
 import { SocketServer } from './utils/socket';
 
-export type { Options };
+export type { Options, InlineOptions };
```

Comment:
How about rename `InlineOptions` to `DevtoolsPluginOptions`?

Reply:
done.

Reply:
not pushed?